platform/ios/CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ Mapbox welcomes participation and contributions from everyone. Please read [CONT
 * Fixed a crash when switching between two styles having layers with the same identifier but different layer types. ([#12432](https://github.com/mapbox/mapbox-gl-native/issues/12432))
 * Fixed an issue where the symbols for `MGLMapPointForCoordinate` could not be found. ([#12445](https://github.com/mapbox/mapbox-gl-native/issues/12445))
 * Fixed an issue causing country and ocean labels to disappear after calling `-[MGLStyle localizeLabelsIntoLocale:]` when the system language is set to Simplified Chinese. ([#12164](https://github.com/mapbox/mapbox-gl-native/issues/12164))
+* Fixed a `CLLocationManager` crash when `MGLMapView` is deallocated. ([#55](https://github.com/mapbox/mapbox-events-ios/pull/55))
```

Comment:
Let's change this entry to reference this actual PR #12542 , and add a reference to https://github.com/mapbox/mapbox-events-ios/pull/55 in a comment.

Reply:
Oops, my mistake! Fixing now.